Runtime/Scripts/RGServiceManager.cs

```diff
@@ -198,6 +203,7 @@ public async Task QueueInstantBot(long botId, Action<RGBotInstance> onSuccess, A
                 },
                 onFailure: async (f) =>
                 {
+                    RGDebug.LogWarning($"Failed to queue instance bot: {f}");
```

Comment:
This is already logged by the caller of this method (but without the id) in `RGOverlayMenu.AddBot`

Reply:
True, it looks like all calling methods handle their errors so I'm going to remove it from this location